src/Helpers/HeaderExtractor.php

```diff
@@ -14,6 +15,13 @@ class HeaderExtractor
     public static function getAuthorizationHeader(HTTPRequest $request)
     {
         $authHeader = $request->getHeader('Authorization');
+        if (!$authHeader) {
+            $envVars = Environment::getVariables();
+            if (isset($envVars['_SERVER']['REDIRECT_HTTP_AUTHORIZATION'])) {
```

Comment:
I believe this should be an .htaccess setting, which should forward the HTTP authorisation. Not something inside the scope of this module?

Reply:
A bit late, but on second thought, I do see value in this, but I'd rather not reed directly from a header, could you update this to use `filter_input`?

Reply:
I suppose I could, shouldn’t be hard. But it’s been a while and I don’t have that setup anymore where this was an issue, so can’t actually test it now. Try to find a moment to set something up...